client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -106,9 +106,29 @@ const SubscriberDataViews = ( {
 		sortTerm: currentView.sort?.field as SubscribersSortBy,
 		sortOrder: currentView.sort?.direction as 'asc' | 'desc',
 		filters: filters,
-		limitData: true,
 	} );
 
+	const {
+		currentSubscriber,
+		onClickUnsubscribe: handleUnsubscribe,
+		onConfirmModal,
+		resetSubscriber,
+	} = useUnsubscribeModal(
+		siteId ?? null,
+		{
+			page: currentView.page ?? 1,
+			perPage: currentView.perPage,
+			search: searchTerm,
+			sortTerm: currentView.sort?.field as SubscribersSortBy,
+			sortOrder: currentView.sort?.direction as 'asc' | 'desc',
+			filters: filters,
+		},
+		false,
+		() => {
+			setSelectedSubscriber( null );
+		}
+	);
```

Comment:
Just moving this next to `useSubscribersQuery` to make it easier to read

Reply:
Because they need to be fed the same parameters to ensure the cache key is correct.